test-record-replay.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
```

Comment:
We cannot run this test on windows without bash. It is best to expose this as a slang-unit-test that uses the Process class to create processes.